pkg/apis/jaegertracing/v1/const.go

```diff
@@ -30,4 +30,7 @@ const (
 
 	// ReconciliationTracer is the OpenTelemetry tracer name for the reconciliation loops
 	ReconciliationTracer string = "operator/reconciliation"
+
+	// CustomMetricsTracer is the OpenTelemetry tracer name for the custom metrics setup
+	CustomMetricsTracer string = "operator/bootstrap-custom-metrics"
```

Comment:
Is there a reason to have this in a different tracer? Why not reuse the bootstrap one, given that this setup is indeed part of the main bootstrap routine?